uutils.yaml

```diff
@@ -18,7 +18,7 @@ environment:
       - ca-certificates-bundle
       - cargo-auditable
       - clang-18-dev
-      - gcc-14-default
+      - gcc
```

Comment:
Unless there's a very good reason, we usually don't explicitly depend on `gcc`, so you can actually remove this line altogether.